examples/miscellaneous/plot_johnson_lindenstrauss_bound.py

```diff
@@ -8,7 +8,8 @@
 dataset can be randomly projected into a lower dimensional Euclidean
 space while controlling the distortion in the pairwise distances.
 
-.. _`Johnson-Lindenstrauss lemma`: https://en.wikipedia.org/wiki/Johnson%E2%80%93Lindenstrauss_lemma
+.. _`Johnson-Lindenstrauss lemma`: https://en.wikipedia.org/wiki/\
```

Comment:
Did this raise a warning?

Reply:
Yea a flake8 warning because I moved it.